api/src/main/java/org/hawkular/apm/api/model/events/SourceInfo.java

```diff
@@ -70,31 +94,31 @@ public void setId(String id) {
     }
 
     /**
-     * @return the sourceUri
+     * @return the fragmentUri
```

Comment:
It represented the URI used to trigger the fragment, so now is actually more accurate.